dtfs-central-api/src/repositories/tfm-deals-repo/tfm-deal-cancellation.repo.deleteOneDealCancellation.test.ts

```diff
@@ -0,0 +1,85 @@
+import { DEAL_SUBMISSION_TYPE, DealNotFoundError, InvalidDealIdError, MONGO_DB_COLLECTIONS, TFM_DEAL_STAGE } from '@ukef/dtfs2-common';
+import { generateAuditDatabaseRecordFromAuditDetails, generateTfmAuditDetails } from '@ukef/dtfs2-common/change-stream';
+import { ObjectId } from 'mongodb';
+import { mongoDbClient as db } from '../../drivers/db-client';
+import { TfmDealCancellationRepo } from './tfm-deal-cancellation.repo';
+import { aTfmUser } from '../../../test-helpers';
+
+const dealId = new ObjectId();
+
+const tfmUserId = aTfmUser()._id;
+
+const auditDetails = generateTfmAuditDetails(tfmUserId);
+
+describe('tfm-deals-cancellation-repo', () => {
+  const updateOneMock = jest.fn();
+  const getCollectionMock = jest.fn();
+
+  beforeAll(() => {
+    jest.useFakeTimers().setSystemTime(1727876437063);
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('deleteOneDealCancellation', () => {
```

Comment:
Should we have this in the parent describe since the file name already mentions deletion

Reply:
I _think_ when we use the same description (i.e. `'tfm-deals-cancellation-repo'`) in two different files, jest can do clever things and group them together in the output (Is this right @AlexBramhill?). I think its a common pattern in dtfs to split tests for one file with multiple exports & keep the describe block the same amongst them all.
Not so big a deal though so i can change it :)

Reply:
Thanks @MarRobSoftwire if this is the existing pattern would prefer to be aligned